ballot.sol.js

```diff
@@ -0,0 +1,68 @@
+var multi = function(func) { return func.toString().match(/[^]*\/\*([^]*)\*\/\}$/)[1]; }
```

Comment:
Ok, so I think it is fine. If it fails, we do not have example code which is not too bad.